config/initializers/assets.rb

```diff
@@ -9,4 +9,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
-Rails.application.config.assets.precompile += %w[print.css]
+Rails.application.config.assets.precompile += %w[print.css test-dependencies.js]
```

Comment:
will this add it to the compiled js in the frontend?

Reply:
Nope, it just means that one of the files built will be a `/assets/collections/test-dependencies-{hash}.js` however that asset would only be included if the HTML references it - which only happens in the test environment.